Update _variables.scss

dariocarbone opened the pull request:
Quick fix for bootstrap 5.3.0

See: 
https://github.com/apalfrey/select2-bootstrap-5-theme/issues/75#issuecomment-1416307961

dariocarbone committed:
Update _variables.scss 

Quick fix for bootstrap 5.3.0

See: 
https://github.com/apalfrey/select2-bootstrap-5-theme/issues/75#issuecomment-1416307961

f1mishutka commented:
Hi,

Replacing $s2bs5-border-color with raw `red` color is not so good solution.

Originally there is `$s2bs5-border-color` variable used. If you trace it to final definition you would have this chain of assignments:
`$s2bs5-border-color = $form-select-border-color = $input-border-color = $border-color`

So correct solution would be to change line 28 of `_variables.scss` from:
```
$s2bs5-border-color:             var(--#{$prefix}form-select-border-color) !default;
```
to
```
$s2bs5-border-color:             $border-color !default;
```

This eliminates compilation bug and does not changes color value in variable.

f1mishutka commented:
Fast solution before this PR is merged is to add this line **before** importing `select2-bootstrap-5-theme`:
```
$s2bs5-border-color: $border-color;
```

dariocarbone committed:
Update _variables.scss

Committed better solution seen here:

https://github.com/apalfrey/select2-bootstrap-5-theme/pull/82#issuecomment-1573261275

dariocarbone commented:
Thanks, you're absolutely right: this is a better solution, promptly committed to master branch

mmarton commented:
Any update on this?

Jir4 commented:
Any news ?

GaetanRole commented:
Up ? It is clearly fast to merge this PR @apalfrey , thanks

wreiske commented:
Looking forward to this! 👍
